### DISCUSSION TITLE
IconLayer different icon style iconAtlas vs auto packing

### DISCUSSION THREAD

**Discussion by radoslavirha** [visgl/deck.gl#7449] [Q&A]
Hi,

I see a difference in icon style between iconAtlas vs auto packing

iconAtlas:
![image](https://user-images.githubusercontent.com/23722745/203599972-84befa7a-65c6-4fe6-a1fe-53affbff1f69.png)

auto packing:
![image](https://user-images.githubusercontent.com/23722745/203599807-356ede21-d12d-4bd2-afb0-0c6d3858fa39.png)

```
new IconLayer({
  ...
  // getIcon: () => ({
  //   url: '.../cross.svg',
  //   width: 1024,
  //   height: 1024,
  //   mask: false
  // }),

  getIcon: () => 'cross',
  iconAtlas: '.../cross.svg',
  iconMapping: {
    cross: { width: 1024, height: 1024, mask: false }
  },

  billboard: false
  ...
})
```

Do you know why it's different? IconAtlas displays correct icon, auto packing is different.

Thanks

**Comment by radoslavirha** [comment]
Maybe important info: I use it in First person view, haven't tried it in Map view

**Reply by Pessimistress** [reply]
Can you share the svg file?

**Reply by radoslavirha** [reply]
Yes, of course.


![cross](https://user-images.githubusercontent.com/23722745/205237344-f3a1a14d-f21a-4f97-8866-5d2ebdcf56c9.svg)
